EZSwiftExtensionsTests/UIColorTests.swift

```diff
@@ -16,59 +16,58 @@ class UIColorTests: XCTestCase {
 	override func setUp() {
 		super.setUp()
 
-		color = UIColor.blueColor()
+		color = UIColor.blue
 	}
 
 	func testInitRGBValues() {
 
 		color = UIColor(r: 255, g: 255, b: 255)
 
-		let red = CGColorGetComponents(color.CGColor)[0]
+		let red = color.cgColor.components?[0]
 		XCTAssertEqual(red, 1)
 	}
 
 	func testInitHexString() {
 
 		color = UIColor(hexString: "ff0000")
-		let red = CGColorGetComponents(color.CGColor)[0]
-		let green = CGColorGetComponents(color.CGColor)[1]
-		let blue = CGColorGetComponents(color.CGColor)[2]
+		let red = color.cgColor.components?[0]
+		let green = color.cgColor.components?[1]
+		let blue = color.cgColor.components?[2]
 
 		XCTAssertEqual(red, 1)
 		XCTAssertEqual(blue, 0)
 		XCTAssertEqual(green, 0)
 	}
 
 	func testInitGray() {
-
 		color = UIColor(gray: 100)
-
-		let red = CGColorGetComponents(color.CGColor)[0]
-		let green = CGColorGetComponents(color.CGColor)[1]
-		let blue = CGColorGetComponents(color.CGColor)[2]
-
+        var red : CGFloat = 0
```

Comment:
@goktugyil maybe

`let red = color.redComponent` (our tested extension) or left as it is